<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FaceAttendence</class>
 <widget class="QMainWindow" name="FaceAttendence">
  <property name="enabled">
   <bool>false</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FaceAttendence</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="camera" native="true">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>480</width>
      <height>480</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>480</width>
      <height>480</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>480</width>
      <height>480</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <widget class="QWidget" name="widget_2" native="true">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>370</y>
       <width>271</width>
       <height>71</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius:15px;
background-color: rgba(49, 50, 54,70);</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="yes">
          <property name="minimumSize">
           <size>
            <width>31</width>
            <height>21</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>31</width>
            <height>21</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">border:none;
background-color: rgba(49, 50, 54,0);
border-image: url(:/yes.png);
</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="success">
          <property name="styleSheet">
           <string notr="true">font: 25 24pt &quot;微软雅黑 Light&quot;;
color: rgb(255, 255, 255);
background-color: rgba(49, 50, 54,0);
border:none;</string>
          </property>
          <property name="text">
           <string>认证成功</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="head_cap_img">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>70</y>
       <width>266</width>
       <height>266</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/crc.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="videoLb">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>480</width>
       <height>480</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <zorder>videoLb</zorder>
    <zorder>widget_2</zorder>
    <zorder>head_cap_img</zorder>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>0</y>
      <width>320</width>
      <height>480</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget {
    border: 1px solid #313236;

	
	background-color: rgba(37, 40, 49,0.9);
}

QWidget#headline{
	
	font: 24pt &quot;微软雅黑&quot;;
	
	color: rgb(255, 255, 255);

}
QWidget#head_image{
	
	border-radius:74px;


}</string>
    </property>
    <widget class="QLabel" name="headline">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>301</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>人脸识别系统</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="head_image">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>70</y>
       <width>150</width>
       <height>150</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>230</y>
       <width>301</width>
       <height>221</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QWidget" name="widget_3" native="true">
        <property name="styleSheet">
         <string notr="true">QWidget {
    border: 1px solid #313236;
    padding: 5px; /* 添加内边距 */
    margin: 1px;  /* 添加外边距 */
	border-radius:4px;
	
	background-color: rgba(37, 40, 49,0.9);
}

QLabel {
    border: none;
    font: bold 12pt &quot;微软雅黑&quot;;
    color: rgb(125, 100, 127);
    
}
QLineEdit {
	border:none;
	
	font: bold 10pt &quot;微软雅黑&quot;;

	color:rgb(255,255,255)

}
</string>
        </property>
        <widget class="QWidget" name="layoutWidget">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>10</y>
           <width>250</width>
           <height>41</height>
          </rect>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>身份</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="LE_identity">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_4" native="true">
        <property name="styleSheet">
         <string notr="true">QWidget {
    border: 1px solid #313236;
    padding: 5px; /* 添加内边距 */
    margin: 1px;  /* 添加外边距 */
	border-radius:4px;
	
	background-color: rgba(37, 40, 49,0.9);
}

QLabel {
    border: none;
    font: bold 12pt &quot;微软雅黑&quot;;
    color: rgb(125, 100, 127);
  
    
}
QLineEdit {
	border:none;
	
	font: bold 10pt &quot;微软雅黑&quot;;

 
	color:rgb(255,255,255)

}
</string>
        </property>
        <widget class="QWidget" name="layoutWidget_4">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>10</y>
           <width>250</width>
           <height>41</height>
          </rect>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>工号</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="LE_Work_ID">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_6" native="true">
        <property name="styleSheet">
         <string notr="true">QWidget {
    border: 1px solid #313236;
    padding: 5px; /* 添加内边距 */
    margin: 1px;  /* 添加外边距 */
	border-radius:4px;
	
	background-color: rgba(37, 40, 49,0.9);
}

QLabel {
    border: none;
    font: bold 12pt &quot;微软雅黑&quot;;
    color: rgb(125, 100, 127);
   
    
}
QLineEdit {
	border:none;
	
	font: bold 10pt &quot;微软雅黑&quot;;

  
	color:rgb(255,255,255)

}
</string>
        </property>
        <widget class="QWidget" name="layoutWidget_7">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>10</y>
           <width>250</width>
           <height>41</height>
          </rect>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_9">
          <item>
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>学院</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_7">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_9" native="true">
        <property name="styleSheet">
         <string notr="true">QWidget {
    border: 1px solid #313236;
    padding: 5px; /* 添加内边距 */
    margin: 1px;  /* 添加外边距 */
	border-radius:4px;
	
	background-color: rgba(37, 40, 49,0.9);
}

QLabel {
    border: none;
    font: bold 12pt &quot;微软雅黑&quot;;
    color: rgb(125, 100, 127);
   
    
}
QLineEdit {
	border:none;
	
	font: bold 10pt &quot;微软雅黑&quot;;

	color:rgb(255,255,255)

}
</string>
        </property>
        <widget class="QWidget" name="layoutWidget_8">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>10</y>
           <width>250</width>
           <height>41</height>
          </rect>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_10">
          <item>
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>姓名</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="LE_Name">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
